src/background.js

```diff
@@ -252,20 +305,41 @@ const rollout = {
     // Determine to skip additional heuristics (by presence of an enterprise policy)
     if (policyEnableDoH === "no_policy_set") {
       // Resetting skipHeuristicsCheck in case a user had a policy and then removed it!
-      this.setSetting("skipHeuristicsCheck", false);
+      this.setSetting("doh-rollout.skipHeuristicsCheck", false);
     } else {
       // Don't check for prefHasUserValue if policy is set to disable DoH
-      this.setSetting("skipHeuristicsCheck", true);
+      this.setSetting("doh-rollout.skipHeuristicsCheck", true);
     }
     return;
 
   },
 
+  async migrateLocalStoragePrefs() {
+    if (await this.getSetting("doneFirstRun")){
```

Comment:
Mmm, eventually I think we'll want `getSetting` to also check the pref in addition to local storage right?

Reply:
Yeah – I think we should/will migrate away from local storage entirely in favor of pref setting.

Reply:
Moved this function to a new PR. #175